lib/compiler-cranelift/src/translator/func_translator.rs

```diff
@@ -243,7 +243,7 @@ fn parse_function_body<FE: FuncEnvironment + ?Sized>(
     // If the exit block is unreachable, it may not have the correct arguments, so we would
     // generate a return instruction that doesn't match the signature.
     if state.reachable {
-        debug_assert!(builder.is_pristine());
+        //debug_assert!(builder.is_pristine());
```

Comment:
What that is for?

Reply:
`builder.is_pristine()` is gone. The replacement is not really equivalent, I removed the assert. I can remove the code entirely instead of just commenting it

Reply:
Ok! Sounds good to me, we should be good to merge